www/include/Administration/parameters/general/help.php

```diff
@@ -69,6 +69,14 @@
 $help['tip_sort_problems_by'] = dgettext('help', 'Default sort in monitoring consoles.');
 $help['tip_order_sort_problems'] = dgettext('help', 'Default order in monitoring consoles.');
 
+/**
+ * Monitoring
+ */
+$help['tip_resource_status_mode'] = dgettext(
+    'help',
+    'Mode to get monitoring information in resource status page.'
```

Comment:
There is no traduction for this ?

Reply:
I dunno if help tooltips are translatable, I'll check and do the translation if needed 👍

Reply:
@a-launois done :)